src/test/java/com/microsoft/azure/spring/data/cosmosdb/repository/integration/ReactiveCourseRepositoryIT.java

```diff
@@ -194,4 +194,22 @@ public void testFindByDepartmentIn() {
             repository.findByDepartmentIn(Collections.singletonList(DEPARTMENT_NAME_2));
         StepVerifier.create(byDepartmentIn).expectNextCount(2).verifyComplete();
     }
+
+    @Test
+    public void testFindAllByPartitionKey() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/Microsoft/spring-data-cosmosdb/pullRequest?prid=5070215)